src/data/drinkRecipes/strawberryKiss.ts

```diff
@@ -0,0 +1,73 @@
+import type { DrinkRecipe } from "types/DrinkRecipe"
+import { DrinkRecipeIngredientType } from "types/DrinkRecipe"
+
+const fakeImageUrl =
+  "https://www.theplunge.com/wp-content/uploads/2020/01/Strawberry-Kiss.jpg"
+
+const strawberryKiss: DrinkRecipe = {
+  name: "Strawberry Kiss",
+  imageUrl: fakeImageUrl,
+  measuredIngredients: [
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Vodka",
+      unit: "oz",
+      unitAmount: 1.5,
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Simple Syrup",
+      unit: "oz",
+      unitAmount: 0.75,
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Measured,
+      ingredient: "Lemon Juice",
+      unit: "oz",
+      unitAmount: 0.75,
+      alternates: [],
+    },
+  ],
+  numberedIngredients: [
+    {
+      type: DrinkRecipeIngredientType.Numbered,
+      ingredient: "Strawberry",
+      amount: 2.0,
+      alternates: [],
+    },
+    {
+      type: DrinkRecipeIngredientType.Numbered,
+      ingredient: "Basil",
+      suffix: "Leaf",
+      amount: 6,
+      alternates: [],
+    },
```

Comment:
Leaf isn't a suffix here; it's a unit. So this should probably be a measured ingredient. I get that it's not "measured", but maybe that name "measuredIngredient" just doesn't fit anymore. This will read "6 Basil Leaf" in the instructions.

This should either be a measured ingredient, or the ingredient should be both words "Basil Leaf" (so that it's puralized for the amount "6" in the instructions and ingredient list, and have `"Basil Leaf": ["Basil"]` in the ingredientToIngredientAliases list.